package Exceptions;

public class USe_Of_Throws {
	public static void main(String[] args) throws Throwable{
		System.out.println("Hello");
		Thread.sleep(2000);
	}

}
